Index.html

```diff
@@ -20,21 +20,140 @@
+    </section>
+    <section class="section">
+        <img class="imgalin" src="./images/Snapshoot3.svg">
+        <h2>Tonic</h2>
+        <span class="sublinks">
+            <span>CANOPY</span>
+            <ul>
+                <li><a href="#">Bank End Dev</a></li> 
+                <li><a href="#">2015</a></li> 
+                          
+             </ul>
+        </span>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <span class="sublinksround">
+            
+            <ul>
+                <li><a href="#">HTML</a></li> 
+                <li><a href="#">CSS</a></li> 
+                <li><a href="#">JavaScript</a></li>          
+             </ul>
+        </span>
+      <div><a href="" class="seeproject">See Project</a></div>
+    </section>
+    <section class="section">
+        <img class="imgalin" src="./images/Snapshoot4.svg">
+        <h2>Multi-Post Stories</h2>
+        <span class="sublinks">
+            <span>CANOPY</span>
+            <ul>
+                <li><a href="#">Bank End Dev</a></li> 
+                <li><a href="#">2015</a></li> 
+                          
+             </ul>
+        </span>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <span class="sublinksround">
+            
+            <ul>
+                <li><a href="#">HTML</a></li> 
+                <li><a href="#">CSS</a></li> 
+                <li><a href="#">JavaScript</a></li>          
+             </ul>
+        </span>
+      <div><a href="" class="seeproject">See Project</a></div>
+    </section>
+    <section class="section">
+        <span>&nbsp;</span>
+        <span>&nbsp;</span>
+        <h2>About Myself</h2>
+        
+        <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+        <span class="sublinksround">
+            
+            <ul>
+                <li><a href="#">HTML</a></li> 
+                <li><a href="#">CSS</a></li> 
+                <li><a href="#">JavaScript</a></li>          
+             </ul>
+        </span>
+     
+    </section>
+    
+    <section class="section">
+        <span class="connect"><a href="#" > LET’S CONNECT</a></span>
+        <ul class="socials">
            <li><a href="#"><img src="./images/twitter.png"></a></li>
            <li><a href="#"><img src="./images/linkedin.svg"></li>
            <li><a href="#"><img src="./images/m.png"></a></li>
            <li><a href="#"><img src="./images/git.png"></a></li>
            <li><a href="#"><img src="./images/peace.png"></a></li>
         </ul>
-    </footer>
+        <div><a href="" class="seeproject">Get my resume</a></div>
+    </section>
+    <section class="section">   
+        <h3><img src="./images/lang.png"></h3>
```

Comment:
- [ ] Please as requested by previous reviewers, kindly add the drop-down arrow icon to the right of languages. You are expected to stick to the design template as much as possible.
### Your design
![image](https://user-images.githubusercontent.com/94551484/227714635-175fdd13-17ae-4fc5-ad29-14d2628a9469.png)
### Figma template
![image](https://user-images.githubusercontent.com/94551484/227714666-1f7f8c2b-0bf5-4912-997b-3099d4b54262.png)